src/include/duckdb/common/types/vector_buffer.hpp

```diff
@@ -49,9 +75,20 @@ class VectorBuffer {
 
 	static buffer_ptr<VectorBuffer> CreateStandardVector(PhysicalType type);
 	static buffer_ptr<VectorBuffer> CreateConstantVector(PhysicalType type);
+	static buffer_ptr<VectorBuffer> CreateConstantVector(VectorType vectorType, const LogicalType &logicalType,
+	                                                     PhysicalType type);
+	static buffer_ptr<VectorBuffer> CreateStandardVector(VectorType vectorType, const LogicalType &logicalType,
+	                                                     PhysicalType type);
+	static buffer_ptr<VectorBuffer> CreateStandardVector(VectorType vectorType, PhysicalType type);
 
-protected:
+public:
```

Comment:
Same here, please use a getter and setter to avoid making member variables public.

Reply:
Sure!